src/sigmoid.js

```diff
@@ -1,18 +1,13 @@
 'use strict';
 
-import {Tensor} from './tensor.js';
+import {unary} from './unary.js';
 
 /**
  * Compute the sigmoid function of the input tensor.
+ * The calculation follows the expression 1 / (exp(-x) + 1).
  * @param {Tensor} input
  * @return {Tensor}
  */
 export function sigmoid(input) {
-  const output = new Tensor(input.shape);
-  for (let i = 0; i < input.size; ++i) {
-    const x = input.getValueByIndex(i);
-    const y = 1 / (Math.exp(-x) + 1);
-    output.setValueByIndex(i, y);
-  }
-  return output;
+  return unary(input, (x) => 1 / (Math.exp(-x) + 1));
```

Comment:
Here I tried to invoke `exp` of `unary.js` likes following
```js
import {add, div} from './binary.js';
import {Tensor} from './tensor.js';
import {exp, neg, unary} from './unary.js';

export function sigmoid(input) {
  const TENSOR1 = new Tensor([1], [1]);
  return div(TENSOR1, add(exp(neg(input)), TENSOR1));
}
```

these codes look not simple, so I still use `Math.exp`, and same to implementation of `tanh`.

@huningxin What's your opinion?

Reply:
Using Math.exp looks good.